### ISSUE TITLE
Support evaluating model on validation data at the end of each epoch

### ISSUE THREAD

**Issue by IaroslavElistratov** [themis-ai/capsa#37]
Support `validation_split` and `validation_data` inside `model.fit()`. Currently observe the following behavior when specifying these parameters: `2/2 [========] - 0s 80ms/step - mve_loss: 0.1286 - val_loss: 0.0000e+00`

Need to implement `test_step` in each metric wrapper.

**Comment by IaroslavElistratov** [comment]
Closing as completed.
